app/controllers/user_controller.rb

```diff
@@ -110,42 +110,19 @@ def delete_theme
     redirect_to edit_user_profile_path
```

Comment:
I expected a redirect to the edit_user_theme_path

Reply:
Fixed!